ftp/src/main/scala/akka.stream.contrib.ftp/FtpSource.scala

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.contrib.ftp
+
+import java.net.InetAddress
+import akka.stream.{ Attributes, Outlet, SourceShape }
+import akka.stream.Attributes.{ InputBuffer, name }
+import akka.stream.contrib.ftp.FtpConnectionSettings.{ BasicFtpConnectionSettings, DefaultFtpConnectionSettings, DefaultFtpPort }
+import akka.stream.contrib.ftp.FtpCredentials.{ AnonFtpCredentials, NonAnonFtpCredentials }
+import akka.stream.scaladsl.Source
+import akka.stream.stage.{ GraphStageLogic, GraphStageWithMaterializedValue, OutHandler }
+import org.apache.commons.net.ftp.{ FTPClient, FTPListParseEngine }
+import scala.concurrent.{ Future, Promise }
+import scala.util.control.NonFatal
+
+object FtpSource {
+
+  final val SourceName = "ftpSource"
+
+  def apply(): Source[FtpFile, Future[Long]] = apply(DefaultFtpConnectionSettings)
```

Comment:
remove this, does not make much sense to have default API bind to localhost IMO, rather make people spell it out.